daedalus/src/Daedalus/ClientApi.purs

```diff
@@ -22,8 +22,8 @@ import Daedalus.Crypto as Crypto
 getProfile :: forall eff. Eff (ajax :: AJAX | eff) (Promise Json)
```

Comment:
I think it would be ok to leave it as is, but ok. Now it's going to return a string.